incubator/timescale/ci/test-values.yaml

```diff
@@ -1,2 +1 @@
-loadBalancer:
-  enabled: false
+
```

Comment:
I think we want to keep the old version of this file, no?

Reply:
I removed as it was failing on lint

Reply:
```
>>> helm lint incubator/timescale --values incubator/timescale/ci/test-values.yaml
==> Linting incubator/timescale
[INFO] Chart.yaml: icon is recommended
[ERROR] values.yaml: - (root): Additional property loadBalancer is not allowed
```